docs/readme/de-DE/docs/contribute/developer-guide.md

```diff
@@ -130,7 +130,13 @@ APITable hat die Debug-Konfigurationen dieser beiden IDEs vorbereitet.
 Wir haben zwei Möglichkeiten, die Übersetzung von APITable zu verbessern:
 
 1. Sie können die Markdown-Dateien im Quellcode ändern und direkt eine PR-Datei erstellen
-2. Treten Sie unserem [Crowdin](https://crowdin.com/project/apitablecode) bei, um die `strings` zu finden, um sie zu ändern
+2. Join our [Crowdin](https://crowdin.com/project/apitablecom) to find the `strings` to modify
```

Comment:
Wrong language?

Reply:
The English source file has changed, and Crowdin will replace all other languages of the strings with English, and then prompt the collaborator to re translate the strings

Reply:
The link address here has changed, so prompt for re translation.